README.md

```diff
@@ -37,143 +37,160 @@ pre-commit install
 
 ## Basic usage
 
-The package main component is a python object called `SDK` which can be used to create a `Batch` and send it automatically
-to Pasqal APIs using an API token generated in the [user portal](https://portal.pasqal.cloud).
 
-A `Batch` is a group of jobs with the same sequence that will run on the same QPU. For each job of a given batch you must set a value for each variable, if any, defined in your sequence.
-The batch sequence can be generated using [Pulser](https://github.com/pasqal-io/Pulser). See their [documentation](https://pulser.readthedocs.io/en/stable/),
-for more information on how to install the library and create your own sequence.
+### Authentication
 
-Once you have created your sequence, you should serialize it as follows:
+There are several ways to provide a correct authentication using the SDK.
```

Comment:
Suggestion: `There are several ways to provide a correct authentication using the SDK.` -> `There are several ways to authenticate using the SDK.`